distribution/command-line/src/main/kotlin/org/codice/ckt/Command.kt

```diff
@@ -24,7 +24,7 @@ fun main(args: Array<String>) {
     val arguments = parser.parse(args)
 
     val idpMetadata = arguments.option("i")
-            ?: "$samlDist/conf/idp-metadata.xml"
+            ?: "$samlDist/conf/test-idp-metadata.xml"
```

Comment:
❗️ This should remain `idp-metadata` because it is not the test's IdP metadata, it is the metadata of the IdP we are testing. 

We will have a `test-idp-metadata` when we start the SP tests to represent our IdP. I agree it might be confusing. We can rename our sp metadata from `test-sp-metadata` to `samlconf-sp-metadata` maybe.